packages/mdc-select/mdc-select.scss

```diff
@@ -27,12 +28,15 @@
 // postcss-bem-linter: define select
 .mdc-select {
   @include mdc-typography-base;
+  @include mdc-states;
```

Comment:
This should be in `--box` styles where the other ripple mixins are used. It doesn't make sense for this to apply to anything without those styles.

When you move it, change it to: `@include mdc-states(text-primary-on-light, $has-nested-focusable-element: true);`

This will make the ripple match the ink color, and make focus shade work properly given the nested select element, even when ripple JS isn't used (I know, right now we require JS, but this seems like the more technically correct thing to do esp. if there is ever hope of CSS-only down the road)